cheatsheets/Docker_Security_Cheat_Sheet.md

```diff
@@ -133,14 +133,13 @@ spec:
 
 As a Kubernetes cluster administrator, you can refer to Kubernetes documentation to configure it using [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
 
-### RULE \#5 - Disable inter-container communication (--icc=false)
+### RULE \#5 - Be mindful of Inter Container Connectivity
```

Comment:
```suggestion
### RULE \#5 - Be mindful of Inter-Container Connectivity
```
I think a hyphen is called for here as well. It's hyphenated in the [docs](https://docs.docker.com/network/drivers/bridge/#connection-limit-for-bridge-networks)